From 797aaa00701c833ad383adf22eb8dd18cb312f42 Mon Sep 17 00:00:00 2001
From: Todd Whiteman <todd.whiteman@joyent.com>
Date: Tue, 19 Feb 2019 16:45:05 -0800
Subject: [PATCH] TRITON-1242 Create DAPI filter to allow filtering out virtual
 servers Reviewed by: Trent Mick <trentm@gmail.com> Approved by: Trent Mick
 <trentm@gmail.com>

---
 package.json | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/package.json b/package.json
index f803b9e..b6908c9 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "vmapi",
   "description": "VMs API",
-  "version": "9.7.0",
+  "version": "9.7.1",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
@@ -28,7 +28,7 @@
     "strsplit": "1.0.0",
     "trace-event": "1.3.0",
     "triton-metrics": "0.1.1",
-    "triton-tags": "1.2.2",
+    "triton-tags": "1.3.0",
     "vasync": "2.2.0",
     "verror": "1.10.0",
     "wf-client": "0.3.1"
-- 
2.21.0

